### ISSUE TITLE
No latest stable release for Linux

### ISSUE THREAD

**Issue by danielhjacobs** [MicrosoftEdge/EdgeWebDriver#156]
On https://developer.microsoft.com/en-us/microsoft-edge/tools/webdriver/, the Linux driver is missing for 126.0.2592.113. According to https://github.com/actions/runner-images/blob/ubuntu22/20240721.1/images/ubuntu/Ubuntu2204-Readme.md#browsers-and-drivers, GitHub actions uses this version of Edge, and using wdio, we are trying to run tests using EdgeDriver based on the system's installed Edge browser (which is this version) in https://github.com/ruffle-rs/ruffle/actions/runs/10068053329/job/27832705746. This fails for Ubuntu 22.04 since https://msedgedriver.azureedge.net/126.0.2592.113/edgedriver_linux64.zip does not exist.

**Comment by torokati44** [comment]
Yeah, please don't put out half-a*sed releases... No Linux build means the release is not done yet.

**Comment by danielhjacobs** [comment]
To go over the problems that lead to this issue:

The runner-images repo has a [script](https://github.com/actions/runner-images/blob/da684ca6d2b6de512a4dec746c7c134d2b010626/images/ubuntu/scripts/build/install-microsoft-edge.sh#L20) which installs Microsoft Edge stable from https://packages.microsoft.com/repos/edge/pool/main/m/microsoft-edge-stable/. The current Microsoft Edge stable there is 127.0.2651.74.

That [script](https://github.com/actions/runner-images/blob/da684ca6d2b6de512a4dec746c7c134d2b010626/images/ubuntu/scripts/build/install-microsoft-edge.sh#L34-L37) then gets the major version of the installed Edge release and pulls Edgedriver from [https://msedgedriver.azureedge.net/LATEST_RELEASE_${edge_version_major}_LINUX](https://msedgedriver.azureedge.net/LATEST_RELEASE_127_LINUX). The latest 127 release with a Linux version is 127.0.2651.72.

That causes a version mismatch between the installed Edge version and the installed EdgeDriver. Perhaps I can get runner-images to fix it, but that wouldn't be necessary if all the builds released at once.

With that version mismatch, apparently [wdio](https://github.com/webdriverio/webdriverio) fails when running tests on Edge.

**Comment by danielhjacobs** [comment]
I did open https://github.com/actions/runner-images/issues/10336 as well.

**Comment by danielhjacobs** [comment]
I opened https://github.com/webdriverio/webdriverio/issues/13251 as well. I don't really care whether this repo, runner-images, or wdio fix this issue, but someone needs to do so.

**Comment by danielhjacobs** [comment]
This has been circumvented downstream in npm's edgedriver by https://github.com/webdriverio-community/node-edgedriver/commit/68043fef07b47e21056da3bc928110e6a759715e which is included in https://github.com/webdriverio-community/node-edgedriver/releases/tag/5.6.1, so if you use webdriverio with edgedriver you can circumvent this issue by updating the webdriverio-related and edgedriver dependencies. However, it would be better if all the EdgeWebDriver releases included all the build artifacts at all times.
